### PR TITLE
revert todraft action, use editing instead

### PR DESCRIPTION
**Pull request by marinaglancy**
- Instead of "todraft" action, allow to edit archived versions but do not allow to save them as "minor change"
- Added behat for policy management
- Raised issue MDL-61844 - manager is requested to agree to new site policies in the middle of the session

### CODE REVIEW

File: classes/output/page_managedocs_list.php [hunk 1]
```diff
@@ -238,11 +238,10 @@ protected function export_version_for_template($output, $policy, $version, $stat
         }
         if ($status == policy_version::STATUS_ARCHIVED) {
```
Comment (mudrd8mz) [inline classes/output/page_managedocs_list.php:239]: Can't we simply remove the condition on the Edit action block above (around line 202)?

File: classes/form/policydoc.php [hunk 2]
```diff
@@ -90,8 +90,8 @@ public function definition() {
         $mform->addRule('content_editor', null, 'required', null, 'client');
 
         if (!$formdata->id || $formdata->status == policy_version::STATUS_DRAFT) {
-            // Creating a new version or editing a draft.
-            $mform->addElement('hidden', 'minorchange', 1);
```
Comment (mudrd8mz) [inline classes/form/policydoc.php:94]: So how/where is the value of this hidden element set now (after removing the third param)?

**Review by mudrd8mz — COMMENTED** [review]
I really like the solution. Very elegant, very clever. Well done!

File: classes/form/policydoc.php (hunk 2 above)
Comment (marinaglancy) [reply classes/form/policydoc.php:94]: it's part of $formdata , see line 130 of the same file and line 157 of the editpolicydoc.php

File: classes/output/page_managedocs_list.php (hunk 1 above)
Comment (marinaglancy) [reply classes/output/page_managedocs_list.php:239]: the labels are still different

Comment (mudrd8mz) [reply classes/output/page_managedocs_list.php:239]: Oh yes, I see. Thanks.

File: classes/form/policydoc.php (hunk 2 above)
Comment (mudrd8mz) [reply classes/form/policydoc.php:94]: Oh I see now. Thanks.

**Review by mudrd8mz — APPROVED** [review]

**Pull request merged**